### ISSUE TITLE
[FEATURE] NodeJS addon

### ISSUE THREAD

**Issue by elderapo** [reacherhq/check-if-email-exists#1105]
This repo doesn't seem to have discussions open so sorry if I am asking this in the wrong place but here we go...

Would be nice to expose this library for NodeJS users through a native addon (ex. https://github.com/neon-bindings/neon). From what I've gathered this library is pretty self-contained so it doesn't need to be hosted as a service.

**Comment by amaurym** [comment]
True, neon would be a good addition, and can be used in Node.JS instead of WASM: #386.

@elderapo Would you like to tackle this issue? Feel free to publish the NPM package under your name, I'll can then put a link in the README.

**Comment by elderapo** [comment]
Haven't had much experience with rust but will try. Thinking of adding typescript definitions for better DX aswell.

**Comment by followdarko** [comment]
Any updates here?

**Comment by elderapo** [comment]
Unfortunately no, if anyone wants to tackle this issue feel free to do so!

**Comment by sanyamgandhak** [comment]
I will pick this up
